pkgs/development/libraries/libopendmarc/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, libmilter }:
+
+stdenv.mkDerivation rec {
+  pname = "libopendmarc";
+  version = "1.3.3";
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ libmilter ];
```

Comment:
```suggestion
```

Remove. The `--with-milter` flag is necessary, but that makes everything visible to libopendmarc, so it does not have to be added to `nativeBuildInputs` as well.